### ISSUE TITLE
Non-coercion in voter results

### ISSUE THREAD

**Issue by Thetoxicarcade** [mdipierro/evote#27]
If they have a token that gets published, couldn't the vote be coerced?

Obviously they could coerce people at the polling place, but this is essentially metadata spying if they can prove to someone (i.e., google reading the email sent to them) that they know what they voted...

possibly?

**Comment by mdipierro** [comment]
You are correct. The fact is that either you allow people to check their vote is counted correctly (and therefore give them a receipt that can be used to coerce vote) or you don't. You cannot have it both ways. Perhaps evote should have a flag so that one can choose one option over the other. It was required with requirements from the PSF.
